src/Simulation/Simulators/Microsoft.Quantum.Simulators.csproj

```diff
@@ -29,6 +29,15 @@
   </ItemGroup>
 
   <ItemGroup>
+    <!-- This is required for simulators' libraries to be copied to the output folder when this project is directly referenced. -->
+    <None Include="$(QSimDll)">
+      <Link Condition="$([MSBuild]::IsOsPlatform('Windows'))">simulators\win-x64\%(FileName)%(Extension)</Link>
+      <Link Condition="$([MSBuild]::IsOsPlatform('OSX'))">simulators\osx-x64\%(FileName)%(Extension)</Link>
+      <Link Condition="$([MSBuild]::IsOsPlatform('Linux'))">simulators\linux-x64\%(FileName)%(Extension)</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Visible>false</Visible>
+    </None>
+    <!-- This is required to be able to invoke the native simulator through .NET. -->
```

Comment:
I don't think this is true, and we shouldn't need to duplicate the file with two different names into the same project/package. If we use the right names and the right folder structure, then .NET will handle the platform specific loading logic for us.

Reply:
Removed the duplication.